tools/mtouch/Application.cs

```diff
@@ -1068,23 +1085,29 @@ public static void CopyMsymData (string src, string dest)
 
 			var p = new Process ();
 			p.StartInfo.UseShellExecute = false;
-			p.StartInfo.RedirectStandardOutput = true;
 			p.StartInfo.RedirectStandardError = true;
 			p.StartInfo.FileName = "mono-symbolicate";
 			p.StartInfo.Arguments = $"store-symbols \"{src}\" \"{dest}\"";
 
-			if (p.Start ()) {
-				var error = p.StandardError.ReadToEnd();
-				p.WaitForExit ();
-				if (p.ExitCode == 0)
-					return;
-				else {
-					Console.Error.WriteLine ($"Msym files could not be copied from {src} to {dest}: {error}.");
-					return;
+			try {
+				if (p.Start ()) {
+					var error = p.StandardError.ReadToEnd();
+					p.WaitForExit ();
+					if (p.ExitCode == 0)
+						return;
+					else {
+						Console.Error.WriteLine ($"Msym files could not be copied from {src} to {dest}: {error}.");
+						throw new MonoTouchException (95, true, "Aot files could not be copied to the destination directory: {dest}"); 
```

Comment:
Throwing the warning will effectively turn it into an error.

Use `ErrorHelper.Warning` instead, which will just print it.

This also makes the `Console.Error.WriteLine` above redundant.